### DISCUSSION TITLE
Release date for 3.0.0

### DISCUSSION THREAD

**Discussion by jhaand** [algesten/ureq#840] [General]
I'm currently writing an embedded application using ureq to read out some energy measurement devices in my home. After it's finished I guess I will not touch it for a while, so upgrading it now would seem worthwhile. It seems that 3.0.0 has a lot simpler API than 2.10. 

Could you tell me approximately when 3.0.0 will be released?

And what is the simplest way to use 3.0.0rc1 in my current project?

**Comment by algesten** [comment]
> Could you tell me approximately when 3.0.0 will be released?

I don't really know. I got a full time job to juggle with this as well :)

> And what is the simplest way to use 3.0.0rc1 in my current project?

You can use the rc1 from crates.io no?

**Reply by jhaand** [reply]
OK. 

I will try.

**Comment by ShayBox** [comment]
What about replacing any removed methods like `timeout` with deprecated `unimplemented` blocks, directing users to the new way until v3.1?
